docs/wallet-integration-guide/src/exchange-integration/extensions.rst

```diff
@@ -6,16 +6,94 @@ Integration Extensions
 Multi-Hosted Parties
 ~~~~~~~~~~~~~~~~~~~~
 
-Sketch:
-
-* allocate the ``treasuryParty`` from the start as a `decentralized party <https://docs.digitalasset.com/operate/3.4/howtos/operate/parties/decentralized_parties.html>`__ on multiple validator nodes
-
-  * a common setup is to use two nodes with threshold two; thus both of them must confirm a tx for it to be committed
-
-* upload all .dars for your onboarded tokens on all of these validators
-* use any of the nodes hosting the party to read from as part of history ingestion and to prepare and execute transactions
-
-  * switching between the nodes can be done. take care to re-synchronize the ingestion offset as described in :ref:`restore-from-validator-node-backup`
+As part of setting up the ``treasuryParty``, you need to sign the
+``PartyToParticipant`` mapping which defines the set of confirming
```

Comment:
This comes out of the blue at this point. 

Consider turning this around:
refer to the default setup where the treasuryParty is hosted on a single node. The explain the options to use multiple nodes.

Push the `PartyToParticipantMapping` down into the technical how-to.

Reply:
rephrased